typings/index.d.ts

```diff
@@ -3783,7 +3786,7 @@ interface FetchInviteOptions extends BaseFetchOptions {
 }
 
 interface FetchInvitesOptions {
-  channelId?: GuildChannelResolvable;
+  channelId?: TextChannel | VoiceChannel | NewsChannel | StoreChannel | StageChannel | Snowflake;
```

Comment:
Since this type is used more than once, might be better to make a type alias for it